components/button/share/src/index.js

```diff
@@ -0,0 +1,83 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+import {Socialtwitter, Socialfacebook, Commentsquare, Envelopeclosed} from '@schibstedspain/sui-svgiconset'
+
+const getOnClickHandle = src => () => window.open(src)
+
+const parseHttpSchemeOfUrl = (urlToEval) => {
+  const isUrlValid = urlToEval.match(/http:\/\/|https:\/\//)
+  return !isUrlValid ? `http://${urlToEval}` : urlToEval
+}
+
+const getShareDefinition = (url, shareText, type) => {
+  const parsedShareUrl = url && parseHttpSchemeOfUrl(url)
+  switch (type) {
+    case 'facebook':
+      return {
+        src: `https://www.facebook.com/share.php?u=${parsedShareUrl}?opi=140&quote=${shareText}&tti=1&pagination=1&RowGrid=2`,
+        icon: Socialfacebook
+      }
+    case 'twitter':
+      return {
+        src: `http://twitter.com/share?url=${parsedShareUrl}&tti=1&pagination=1&RowGrid=2&text=${shareText}`,
+        icon: Socialtwitter
+      }
+    case 'whatsapp':
+      return {
+        src: `whatsapp://send?text=${shareText} - ${parsedShareUrl}`,
+        icon: Commentsquare
+      }
+    case 'email':
+      return {
```

Comment:
Shouldn't we do a mailto with the shareText and the url as body in this case ??

Reply:
Yup totally agree! Our email in 'obra nueva' opens a modal an we didn't thought about use mailTo as the default value here. Thank you!